samples/app/src/main/webapp/browse.jsp

```diff
@@ -67,7 +67,7 @@
 	</div>
 <ul>
 <li><a href="apps">Apps</a></li>
-<li><a href="j_spring_security_logout">Logout</a></li>
```

Comment:
these didn't quite work still....  originals were wrong, but new ones might not get properly subt'ed still.